paper-card.sublime-snippet

```diff
@@ -0,0 +1,14 @@
+<snippet>
+	<content><![CDATA[
+<paper-card heading="${1:Card title}">
+	<div class="card-content">$2</div>
+	  <div class="card-actions">
```

Comment:
I think the indentation may be off here. Does this need to be a sibling of `.card-content`?